packages/react-flicking/src/react-flicking/Flicking.tsx

```diff
@@ -207,7 +206,10 @@ class Flicking extends React.Component<Partial<FlickingProps & FlickingOptions>>
       flicking.on(eventName, e => {
         e.currentTarget = this;
 
-        props[propName](e);
+        const evtHandler = this.props[propName];
+        if (evtHandler) {
+          evtHandler(e);
+        }
```

Comment:
As all Flicking event handlers have a default value, checking existence (`if (eventHandler)`) can be omitted.
- https://github.com/naver/egjs-flicking/blob/e6a393d3f061141091bac9f2b156970b43e6701e/packages/react-flicking/src/react-flicking/consts.ts#L33-L49